Taskfile.yml

```diff
@@ -106,26 +106,14 @@ tasks:
     generates:
       - "{{ .BUILD_BIN }}"
 
-  build-all:
+  build:
     desc: "Compiles all Lambda handlers"
     deps:
       - build-download_grants_gov_db
-      - build-split_grants_gov_xml_db
 
   build-download_grants_gov_db:
     desc: Compiles download_grants_gov_db
     cmds:
       - task: build-lambda
         vars:
           LAMBDA_CMD: download_grants_gov_db
-    generates:
-      - "./bin/download_grants_gov_db/bootstrap"
-
-  build-split_grants_gov_xml_db:
-    desc: Compiles split_grants_gov_xml_db
-    cmds:
-      - task: build-lambda
-        vars:
-          LAMBDA_CMD: split_grants_gov_xml_db
-    generates:
-      - "./bin/split_grants_gov_xml_db/bootstrap"
```

Comment:
🙈

Reply:
This is from the future and its existence was disrupting spacetime. Stay tuned for the next PR....